src/components/atoms/two-step-delete/index.tsx

```diff
@@ -0,0 +1,99 @@
+import clsx from "clsx"
+import React, {
+  forwardRef,
+  useCallback,
+  useEffect,
+  useImperativeHandle,
+  useRef,
+  useState,
+} from "react"
+import TrashIcon from "../../fundamentals/icons/trash-icon"
+import Spinner from "../spinner"
+
+type Props = {
+  onDelete: () => void
+  deleting?: boolean
+}
+
+const TwoStepDelete = forwardRef<HTMLButtonElement, Props>(
```

Comment:
This button needs a popover saying "Are you sure?". Also the text is different "confirm"/"delete"

| design | implementaiton |
|-|-|
| <img width="221" alt="image" src="https://user-images.githubusercontent.com/88927411/186053863-55d29204-d8f1-47df-a9e0-a30441544fe9.png"> | <img width="169" alt="image" src="https://user-images.githubusercontent.com/88927411/186053950-c46792f7-207f-49e2-8a57-d3bdf4f50ea9.png"> |

Reply:
Great catch, should be good now 🥳